frontend/package-lock.json

```diff
@@ -2768,9 +2768,9 @@
       }
     },
     "callsites": {
```

Comment:
This file probably shouldn't have changes?

Reply:
We have no idea what this would be related to?

Reply:
We should probably reset the changes. You can do this with `git fetch origin staging && get checkout origin/staging -- frontend/package-lock.json`.